### DISCUSSION TITLE
eQ3-HM-LGW als LAN Gateway

### DISCUSSION THREAD

**Discussion by Mathias-Ro** [OpenCCU/OpenCCU#1674] [🇩🇪 Fragen&Antworten]
Hallo,
ich habe mir die aktuelle Version (3.61.7.20220115) auf einem Proxmox-Server mit qm importovf installiert.
Meine Komponenten werden im Normalbetrieb per FHEM mit einem eQ3-HM-LGW gesteuert. Dieses arbeitet im Gegensatz zu den alten runden Gateways auf Port 2000 (nicht 1000). Es ist mir aber nicht gelungen dieses in der Systemsteuerung einzutragen. Nach dem Klick auf [Übernehmen] ist der Eintrag wieder verschwunden.
Aus der Konsole lässt sich das Gateway sowohl über die IP, als auch über die Seriennummer per Ping erreichen, der FHEM-Server wurde vorher gestoppt und das Gateway neu gestartet. Der Key wurde vom FHEM mit übernommen.
Wie lässt sich das Gateway korrekt im RaspberryMatic einrichten?
Danke!

**Comment by Baxxy13** [comment]
Von den alten Gateways und deren Integration habe ich leider keinen Plan.
Zeig doch mal Screenshots und gucke in die Browserkonsole ob da Fehler ausgegeben werden.

Die Größe deiner virtuellen Festplatte hast du, wie [hier bei Punkt 6](https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Proxmox-VE) beschrieben, angepasst?
